### PR TITLE
chore: Update aggregator announcement

### PR DESCRIPTION
**Pull request by binarylogic**
I thought it would be easier to open a new PR. Comments are welcome.

### CODE REVIEW

File: website/content/en/highlights/2021-08-24-vector-aggregator.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
 collected by other agents. The core benefit of having distinct agents and aggregators is that
 you can have a separation of concerns within an observability data processing pipeline. Agents
 can become much “thinner,” in some cases acting as pure pipes that collect from a source and
 ship downstream to aggregators, while aggregators perform the “thicker” work of ensuring that
 the data is scrubbed of sensitive information, properly formatted for downstream consumers,
-sampled to reduce volume, and more.
+sampled to reduce volume, and more. This is especially useful when inserting Vector into
+existing setups.
 
 The `vector-aggregator` chart deploys Vector as a StatefulSet, with the option of installing a
```
Comment (spencergilbert) [inline website/content/en/highlights/2021-08-24-vector-aggregator.md:35]:
```suggestion
The `vector-aggregator` chart deploys Vector as a [StatefulSet][statefulset], with the option of installing a
```

Guessing based on the added line at the bottom?

File: website/content/en/highlights/2021-08-24-vector-aggregator.md [hunk 2]
```diff
@@ -1,32 +1,36 @@
 ---
 date: "2021-08-24"
-title: "Moving the aggregator chart to public beta"
-description: "The Vector aggregator Helm chart is now publicly available"
+title: "Moving the Aggregator Helm Chart to public beta"
```
Comment (spencergilbert) [inline website/content/en/highlights/2021-08-24-vector-aggregator.md:3]: LOL, I lower cased these based on Luc's/Jesse's feedback

**Review by spencergilbert — APPROVED** [review]

File: website/content/en/highlights/2021-08-24-vector-aggregator.md (hunk 2 above)
Comment (binarylogic) [reply website/content/en/highlights/2021-08-24-vector-aggregator.md:3]: Aggregator is a proper Vector role and should be capitalized. The same with Helm and Chart. Curious what @lucperkins thinks.

Comment (spencergilbert) [reply website/content/en/highlights/2021-08-24-vector-aggregator.md:3]: https://github.com/timberio/vector/pull/8800#discussion_r692399148 previous discussion

Comment (jszwedko) [reply website/content/en/highlights/2021-08-24-vector-aggregator.md:3]: I'm of the opinion that we should keep it lower-cased until we are ready to make the change and then make the change site-wide.

**Comment by spencergilbert** [comment]
https://github.com/timberio/vector/pull/8800 picked the changes over

**Pull request closed without merging**